exporter/datadogexporter/config/config.go

```diff
@@ -224,5 +230,9 @@ func (c *Config) Sanitize() error {
 		c.Traces.TCPAddr.Endpoint = fmt.Sprintf("https://trace.agent.%s", c.API.Site)
 	}
 
+	if c.Traces.IgnoreResources == nil {
+		c.Traces.IgnoreResources = []string{}
+	}
+
```

Comment:
A `nil` slice works pretty much like an empty slice, I think we can remove this

Reply:
updated